chord_metadata_service/phenopackets/admin.py

```diff
@@ -12,11 +12,6 @@ class PhenotypicFeatureAdmin(admin.ModelAdmin):
     pass
 
 
-@admin.register(m.Procedure)
-class ProcedureAdmin(admin.ModelAdmin):
-    pass
-
-
 @admin.register(m.HtsFile)
 class HtsFileAdmin(admin.ModelAdmin):
```

Comment:
is HtsFile still around? I thought it was removed since it's different from phenov2's File model

Reply:
removed